src/status_im2/contexts/chat/composer/effects.cljs

```diff
@@ -109,18 +109,15 @@
    {:keys [edit]}]
   (rn/use-effect
    (fn []
-     (let [edit-text        (get-in edit [:content :text])
-           text-value-count (count @text-value)]
+     (let [edit-text (get-in edit [:content :text])]
        (when (and edit @input-ref)
          ;; A small setTimeout is necessary to ensure the statement is enqueued and will get executed
          ;; ASAP.
          ;; https://github.com/software-mansion/react-native-screens/issues/472
          (js/setTimeout #(.focus ^js @input-ref) 250)
          (.setNativeProps ^js @input-ref (clj->js {:text edit-text}))
          (reset! text-value edit-text)
-         (reset! saved-cursor-position (if (zero? text-value-count)
-                                         (count edit-text)
-                                         text-value-count)))))
+         (reset! saved-cursor-position (count edit-text)))))
```

Comment:
Why was this change needed?

Reply:
We keep both values the editing value and composer value before editing
It wasn't really necessary, Because if the text-value-count existed while editing the cursor would move to the text-value-count rather than the edit-text-count.